networkx/algorithms/dag.py

```diff
@@ -30,6 +30,8 @@
     "dag_longest_path",
     "dag_longest_path_length",
     "dag_to_branching",
+    "compute_v_structures",
+    "moralize_graph",
```

Comment:
Would the `moralize_graph` and `compute_v_structures` be better housed in the `moral.py` module?  That module only has a single function in it. Maybe it should be brought into `dag.py`? 

I bring it up because this is a good time to consider where these should go. In the end it doesn't really matter, but it would be good to make the code easily discoverable. You know more about this than I do -- what is a good organization of this code?

Reply:
OH didn't even see the moral.py. Oh yeah, perhaps computing v-structures could go in there! I'll sub out the existing moralize_graph function too.

Reply:
Refactored to place them inside `moral.py` now.

Reply:
Actually it doesn't make sense to have them inside `moral.py`, but I would suspect it possibly makes sense to move `moral_graph()` into `dag.py` because I don't see any reason to compute the moral graph on directed graphs with loops... but I could be wrong.